core/com.b2international.snowowl.core/src/com/b2international/snowowl/core/bundle/Bundle.java

```diff
@@ -89,6 +90,10 @@ public Builder toDocumentBuilder() {
 				.bundleId(getBundleId())
 				.settings(getSettings());
 	}
+
+	public static ResourceURI uri(String bundleUri) {
+		return bundleUri.startsWith(RESOURCE_TYPE) ? new ResourceURI(bundleUri) : ResourceURI.of(RESOURCE_TYPE, bundleUri);
```

Comment:
We might want to add a trailing `/` to the constant in these `startsWith` checks. As an example, `"bundlesort"` would be a resource-type-less URI, but the code treats it as one that has a type because it starts with `"bundles"`.